protocol/lavasession/consumer_session_manager_test.go

```diff
@@ -28,45 +30,75 @@ const (
 	firstEpochHeight                   = 20
 	secondEpochHeight                  = 40
 	cuForFirstRequest                  = uint64(10)
-	grpcListener                       = "localhost:28353"
 	servicedBlockNumber                = int64(30)
 	relayNumberAfterFirstCall          = uint64(1)
 	relayNumberAfterFirstFail          = uint64(1)
 	latestRelayCuAfterDone             = uint64(0)
 	cuSumOnFailure                     = uint64(0)
 )
 
+// This variable will hold grpc server address
+var grpcListener = "localhost:28353"
```

Comment:
dont set a specific port, use 0 to make it dynamic so it doesn't fail

Reply:
Fixed f702f884e44c4a1fd4b45f720f3a1675d8c9c158